CMakeLists.txt

```diff
@@ -18,6 +18,7 @@ project(
   VERSION 0.17.0
   LANGUAGES CXX)
 
+include(cmake/Modules/ccache.cmake)
```

Comment:
Can this be guarded by a CMake option? Changing the CMake compiler launcher will break anyone using ccache compiler rsymlinks.

Reply:
Done.